docs/RuntimeAttributes.md

```diff
@@ -186,46 +220,67 @@ runtime {
 }
 ```
 
-## `bootDiskSizeGb`
+### `docker`
+
+When specified, Cromwell will run your task within the specified Docker image. 
 
-In addition to working disks, Google Cloud allows specification of a boot disk size. This is the disk where the docker image itself is booted (**not the working directory of your task on the VM**).
-Its primary purpose is to ensure that larger docker images can fit on the boot disk.
 ```
 runtime {
-  # Yikes, we have a big OS in this docker image! Allow 50GB to hold it:
-  bootDiskSizeGb: 50
+  docker: "ubuntu:latest"
 }
 ```
 
-Since no `local-disk` entry is specified, Cromwell will automatically add `local-disk 10 SSD` to this list.
+- Local: Cromwell will automatically run the docker container.
+- SFS: When a docker container exists within a task, the `submit-docker` method is called. See the [Getting started with containers](/tutorials/Containers/) guide for more information.
+- GCP: This attribute is mandatory when submitting tasks to Google Cloud.
+- AWS Batch: This field is mandatory when submitting tasks to AWS Batch.
```

Comment:
Normalize on `attribute`/`field` on these two entries

Reply:
Removed all references to field in favour of `attribute` here: a86b07e248163b124780141b4c89df0d99d21051